superset/db_engine_specs/hive.py

```diff
@@ -179,6 +171,7 @@ def convert_to_hive_type(col_type: str) -> str:
             bucket_path,
             os.path.join(upload_prefix, table_name, os.path.basename(filename)),
         )
+        # TODO(bkyryliuk): support other cvs delimiters
```

Comment:
typo: `other csv delimiters` 

although with CSV meaning "comma separated values", this might be better as `support other delimiters`